test/.stats-app/stats-config.js

```diff
@@ -32,9 +32,13 @@ const clientGlobs = [
     globs: ['fetched-pages/**/*.html'],
   },
   {
-    name: 'Edge SSR Page bundle Size',
+    name: 'Pages edge SSR bundle Size',
     globs: ['.next/server/pages/edge-ssr.js'],
   },
+  {
+    name: 'App edge SSR bundle Size',
+    globs: ['.next/server/app/app-edge-ssr/page.js'],
+  },
```

Comment:
Looks like the `configFiles` field needs to be updated as well to enable the `app` dir